libsemanage/tests/test_ibendport.c

```diff
@@ -362,6 +373,8 @@ void test_ibendport_modify_del_query_local(void)
 						 &ibendport_local) < 0);
 
 	/* cleanup */
+	semanage_ibendport_key_free(key);
+	semanage_ibendport_free(ibendport_local);
```

Comment:
@cgzones This `semanage_ibendport_free(ibendport_local);` should probably be moved after `CU_ASSERT_PTR_NOT_NULL_FATAL(ibendport_local);`, as the variable is then used again in the test which checks that `semanage_ibendport_query_local` fails,
```c
	CU_ASSERT(semanage_ibendport_query_local(sh, key,
						 &ibendport_local) < 0);
```